ampligraph/datasets/abstract_dataset_adapter.py

```diff
@@ -89,7 +89,7 @@ def set_filter(self, filter_triples):
         """
         raise NotImplementedError('Abstract Method not implemented!')
         
-    def get_next_batch(self, batches_count=-1, dataset_type="train"):
+    def get_next_batch(self, batches_count=-1, dataset_type="train", use_filter=False):
```

Comment:
Was going to suggest this change but you beat me to it :)

Reply:
I think you did mention it on the issue.